api/services/v1alpha1/gateway_types.go

```diff
@@ -27,8 +27,9 @@ const (
 	GatewayServiceName = "gateway"
 	// GatewayInstanceName the name of the GatewayConfig instance singleton.
 	// value should match whats set in the XValidation below
-	GatewayInstanceName = "default-gateway"
-	GatewayConfigKind   = "GatewayConfig"
+	GatewayConfigName  = "default-gateway"
+	GatewayConfigKind  = "GatewayConfig"
+	DefaultGatewayName = "data-science-gateway" // DefaultGatewayName is the default name of the Gateway.
```

Comment:
Since DefaultGatewayName isn’t part of any spec in gateway_types.go, is it correct to keep it here? I feel it should instead belong in tests/e2e/gateway_test.go.

Reply:
yep, it should be removed, it has been added as a const in the gateway package